### ISSUE TITLE
Why is C not supported?

### ISSUE THREAD

**Issue by flyingdutchpig** [bytedance/SandboxFusion#24]
Thanks for this awsome project. I wonder why C language is not supported, is there a special reason?

**Comment by flyingdutchpig** [comment]
Does it mean I have to pass "cpp" for language name in post body for running C code?

**Comment by ssine** [comment]
Yes, cpp is a superset of c so we generally use cpp instead. If you do need a c language to filter out cpp code, open a feature request.
